mautic/mautic
# Plugin permission bug fix

Previously, permissions created by plugins would cause an error when saved because generatePermissions function did not set pluginBundle flag properly when calling getPermissionObject.

To reproduce the issue: 
- Use the helloworld plugin included the [Developer document](https://developer.mautic.org/#creating-custom-permissions).
- Install the plugin and navigate to Roles -> Add new role
- Go to your plugin's permission and select some the click save
- An error page should appear.

To verify fix:
- Apply the changes in this PR and repeat above process. There should no longer be an error after saving permissions. 

Link to related forum topic, including the stack trace after the error is encountered: https://www.mautic.org/community/index.php//1042-custom-permission-plugin
